packages/edit-post/src/components/secondary-sidebar/inserter-sidebar.js

```diff
@@ -36,12 +36,10 @@ export default function InserterSidebar() {
 		focusOnMount: null,
 	} );
 
-	const inserterContentRef = useRef();
+	const libraryRef = useRef();
 	useEffect( () => {
-		inserterContentRef.current
-			.querySelector( '.block-editor-inserter__search input' )
-			.focus();
-	}, [] );
+		libraryRef.current.focusSearch();
+	} );
```

Comment:
I think we need to provide an empty array as the dependency.
```suggestion
	}, [] );
```